ui/src/app/extension/extensions-forms/extension-form-opc.tpl.html

```diff
@@ -212,8 +212,8 @@
                                                     </md-input-container>
 
                                                     <section class="dropdown-section">
-                                                        <div class="tb-container" ng-class="{'ng-invalid':!server.keystore.file}">
-                                                            <span ng-init='fieldsToFill = {"fileName":"fileName", "file":"file"}'></span>
+                                                        <div class="tb-container" ng-class="{'ng-invalid':!server.keystore.location}">
+                                                            <span ng-init='fieldsToFill = {"fileName":"fileName", "location":"location"}'></span>
```

Comment:
Can we keep using 'location' for the file name (as it was before) and 'fileContent' for file content?

Reply:
yeah, will be changed.